apps/condo/bin/migrate-user-import-id-data-to-user-external-identity.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Migrate User.importId/User.importRemoteSystem fields data to separate entity: UserExternalIdentity
+ *
+ * Usage:
+ *      yarn workspace @app/condo node bin/migrate-user-import-id-data-to-user-external-identity
+ */
+
+const path = require('path')
+const dayjs = require('dayjs')
+
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+
+const { User, UserExternalIdentity } = require('@condo/domains/user/utils/serverSchema')
+const { IDP_TYPES } = require('@condo/domains/user/constants/common')
+
+
+// init utils & common vars
+const dv = 1
+const sender = { dv, fingerprint: 'migrate-user-importId-to-externalIdentity' }
+
+const log = (args) => {
+    console.log(`${new Date().toISOString()}: ${JSON.stringify(args)}`)
+}
```

Comment:
Could `getLogger` fit your case?

Reply:
I don't think so. This is one time task - migrate data and I prefer to get all logs directly into the terminal console (exactly like in other one time tasks)